lib/datasource/nuget/get.js

```diff
@@ -1,21 +0,0 @@
-const got = require('../../util/got');
-const hostRules = require('../../util/host-rules');
-
-module.exports = get;
-
-function get(url, options) {
-  const finalOptions = options || {};
-  const hostRule = hostRules.find({ platform: 'nuget', endpoint: url });
```

Comment:
By removing this you changed the behavior for the NuGet data source a bit. The new `utils/got/host-rules` logic only searches host rules by host, and not by endpoint. Having it search by endpoint allows use cases where different keys are required for different endpoints on the same host. I'd now expect those scenarios to be broken. Maybe it'd be better to search by endpoint in `utils/got/host-rules`?

Reply:
@fgreinacher 	sorry for that! Wasn't my intention to change functionality, so I'll see if we can add it back in the new file

Reply:
By the way do you have an active use case for this though? I just looked at the source and found it only reads platform and host: https://github.com/renovatebot/renovate/blob/ca66b34aaaa893fde5e2eb2567a1f4a3a46063d6/lib/util/host-rules.js#L68

Reply:
Hm, I explicitly changed the NuGet data source to use the endpoint because I thought it would be used as a search key, now I'm wondering how that ever worked ❓